src/main/java/com/ioet/bpm/people/domain/AbstractEntity.java

```diff
@@ -0,0 +1,26 @@
+package com.ioet.bpm.people.domain;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAutoGenerateStrategy;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAutoGeneratedTimestamp;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.data.annotation.CreatedDate;
+
+
+import javax.annotation.PostConstruct;
+import java.sql.Timestamp;
+
+@Getter
+@Setter
+public abstract class AbstractEntity {
```

Comment:
I think we can find another name for this class in order to make it clear its intention.

Reply:
I think we agree this class will be renamed as AuditLog, right?